installer/conf/omsagent.d/patch_management.conf

```diff
@@ -23,4 +23,29 @@
 <filter oms.patch_management_immediate_run>
   type filter_patch_management
   log_level warn
+</filter>
+
+<source>
+    @type tail
+    tag oms.update_progress
+    path /var/log/apt/history.log
+    pos_file /var/opt/microsoft/omsagent/run/history.log.pos
+    keep_time_key true
+    format multiline
+    format_firstline /Start-Date:/
+    format1 /Start-Date: (?<start-date>.*)\n/
+    format2 /Commandline: (?<apt-logs>.*\n)/
+    read_from_head true
+    log_level trace
+</source>
+
+<filter oms.update_progress>
+    @type grep
+    regexp1 apt-logs Requested
```

Comment:
The log adds "Requested-By" explicitely when executed by a user (In our case, it'll be the user under which the patching runbook would be run).

Like in the following logs - the Requested-by shows up.
```
Start-Date: 2016-09-13  23:40:15
Commandline: apt install python-pip
Requested-By: varad (1000)
Install: python2.7-dev:amd64 (2.7.12-1~16.04, automatic), libexpat1-dev:amd64 (2.1.0-7ubuntu0.16.04.2, automatic), libpython-all-dev:amd64 (2.7.11-1, automatic), libpython2.7:amd64 (2.7.12-1~16.04, automatic), libpython2.7-dev:amd64 (2.7.12-1~16.04, automatic), libpython-dev:amd64 (2.7.11-1, automatic), python-pkg-resources:amd64 (20.7.0-1, automatic), python-all:amd64 (2.7.11-1, automatic), python-dev:amd64 (2.7.11-1, automatic), python-setuptools:amd64 (20.7.0-1, automatic), python-wheel:amd64 (0.29.0-1, automatic), python-pip-whl:amd64 (8.1.1-2ubuntu0.2, automatic), python-pip:amd64 (8.1.1-2ubuntu0.2), python-all-dev:amd64 (2.7.11-1, automatic)
End-Date: 2016-09-13  23:40:28

Start-Date: 2016-09-13  23:42:33
Commandline: apt-get install -y nodejs
Requested-By: varad (1000)
Install: rlwrap:amd64 (0.41-1build1, automatic), nodejs:amd64 (6.5.0-1nodesource1~xenial1)
End-Date: 2016-09-13  23:42:48

Start-Date: 2016-08-26  13:08:47
Commandline: apt install libopenscap8
Requested-By: varad (1000)
Install: libcurl3:amd64 (7.47.0-1ubuntu2.1, automatic), libopenscap8:amd64 (1.2.8-1), libxslt1.1:amd64 (1.1.28-2.1, automatic)
End-Date: 2016-08-26  13:08:52
```